homeassistant/components/downloader.py

```diff
@@ -62,6 +64,8 @@ def do_download():
 
                 subdir = service.data.get(ATTR_SUBDIR)
 
+                friendly_name = service.data.get(ATTR_FILENAME)
```

Comment:
Can you just call it filename ? Friendly name is confusing as it's an attribute on entities.

Reply:
(just add `filename is None` to the content disposition if check)

Reply:
Sure @balloob. I'll update it later tonight. :+1: